google/cloud/aiplatform/initializer.py

```diff
@@ -130,6 +138,8 @@ def init(
             self._credentials = credentials
         if encryption_spec_key_name:
             self._encryption_spec_key_name = encryption_spec_key_name
+        if network:
```

Comment:
To test against `None` default values, use `is not None` instead.  So here this should be `if network is not None: ...`.

Reply:
Fixed